R/layers2traces.R

```diff
@@ -622,6 +622,67 @@ to_basic.GeomQuantile <- function(data, prestats_data, layout, params, p, ...){
   dat
 }
 
+transform_geomroc <- function(data, n.cuts, size, round, alpha, ...){
+  labels <- as.integer(seq(1, nrow(data), length.out = min(n.cuts, nrow(data), na.rm=T)))
+  text <- data[labels, ]
+  
+  text$label <- as.character(round(text$label, round))
+  text$size <- size
+  text$alpha <- alpha
+  text
+}
+
+#' @export 
+to_basic.GeomRoc <- function(data, prestats_data, layout, params, p, ...){
+  
+  data$alpha <- params$linealpha
+  if(params$n.cuts) {
```

Comment:
If this value is falsy, then `points` won't exist, which will lead to an error on line 660, so why have the `if` statement in the first place?